projects/components/src/utils/test/datagrid/filter-utils.ts

```diff
@@ -62,11 +69,15 @@ export function createDatagridFilterTestHelper<V, C>(
     configureTestingModule();
 
     // Add the filter to grid column
-    const finder = new WidgetFinder(FilterTestHostComponent);
+    const wf = new WidgetFinder(FilterTestHostComponent);
+
+    const finder = new AngularWidgetObjectFinder(FilterTestHostComponent);
```

Comment:
I don't think we should use both `WidgetFinder` and `AngularWidgetFinder`, can't we just use `AngularWidgetFinder`?